asciidoc/workloads/azure/backend/netcore/quickstart/web_api/create_project_netcore.adoc

```diff
@@ -31,7 +31,7 @@ Access Amido.Stacks.Templates package page in Nuget https://www.nuget.org/packag
 .Run the command to install the package
 [source, bash]
 ----
-dotnet new --install Amido.Stacks.Templates
+dotnet new install Amido.Stacks.Templates
```

Comment:
This suggests you're using .NET 7+

https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-new#description

Reply:
I would argue that a .NET developer will / should have .NET 7+ on their local machine.